ports/shader-slang/portfile.cmake

```
@@ -75,6 +75,26 @@ if(VCPKG_SHADER_SLANG_UPDATE)
 	message(FATAL_ERROR "Stopping due to VCPKG_SHADER_SLANG_UPDATE being enabled.")
 endif()
 
+file(GLOB slang_cmake_configs
+		"${BINDIST_PATH}/cmake/*.cmake"
+)
+
+# install cmake files
+set(SLANG_INSTALL_CMAKE_PATH "${CURRENT_PACKAGES_DIR}/share/slang")
+file(INSTALL "${slang_cmake_configs}" DESTINATION "${SLANG_INSTALL_CMAKE_PATH}")
```

I am sorry, I failed on the result variable of the glob :facepalm:
```suggestion
file(INSTALL ${slang_cmake_configs} DESTINATION "${SLANG_INSTALL_CMAKE_PATH}")
```

---

resolved